docs/integrations/overview.mdx

```diff
@@ -18,6 +18,12 @@ description: Learn about the available integrations with Clerk.
 
   ---
 
+  - [Instant](/docs/integrations/databases/instant)
```

Comment:
It doesn't appear as if this link exists

Reply:
(Sorry team, still hacking it up and wanted to view the PR on Github. Will update as soon as it's ready!)

Reply:
Note: PR is updated 🫡